### PR TITLE
Refine design

### PR DESCRIPTION
**Pull request by iojcde**

### CODE REVIEW

File: pages/blog/[slug].tsx [hunk 1]
```diff
@@ -32,41 +32,74 @@ const PostLayout: NextPage<{ post: Post }> = ({ post }) => {
...
+              height="540"
+              quality={100}
+              className="rounded-md object-cover object-center "
+              alt=""
+              priority
+            />
+          </a>
+          <div className="pb-2 text-neutral-600 dark:text-neutral-200">
+            Author
+          </div>
```
Comment (Edqe14) [inline pages/blog/[slug].tsx:76]: I don't think this is required tbh

image

Updated:
- Also decreased the image margin to `1rem`

image

File: components/Nav.tsx [hunk 2]
```diff
@@ -134,193 +134,193 @@ const Nav: React.FC = () => {
     <>
       <a
         href="#content"
-        className="text-md absolute inset-x-0 -top-7 z-50 mx-auto w-40 -translate-y-12 transform rounded-lg border border-gray-300 bg-white px-3 py-2 text-center tracking-tight outline-none transition-transform duration-100 focus:translate-y-12 dark:border-neutral-700 dark:bg-black"
+        className="text-md absolute inset-x-0 -top-7 z-[120] mx-auto w-40 -translate-y-12 transform rounded-lg border border-gray-300 bg-white px-3 py-2 text-center font-semibold tracking-tight shadow-xl outline-none transition-transform duration-100 focus:translate-y-12 dark:border-neutral-700 dark:bg-black"
       >
         Skip to content
       </a>
+      <nav className="fixed left-0 top-0 right-0 z-[150] flex h-[74px] items-center border-b bg-white/80 backdrop-blur-[14px] transition dark:bg-primary/10">
```
Comment (Edqe14) [inline components/Nav.tsx:141]: On dark theme, maybe could make the navbar more opaque compared to the light theme one

image

Light theme

image

**Review by Edqe14 — APPROVED** [review]
Overall LGTM

File: components/Nav.tsx (hunk 2 above)
Comment (iojcde) [reply components/Nav.tsx:141]: One problem that occured when I did that is that the color looked off from the rest of the background...

I'll still try to make it more consistent.

**Pull request merged**